lib/ui/screens/auth.dart

```diff
@@ -9,213 +9,259 @@ Author: Eric Boysen
+                                                                  return 'Please enter a valid email or phone';
+                                                                }
+                                                                return null;
+                                                              },
+                                                            ),
+                                                            Text('Password'),
+                                                            CustomField(
+                                                              hintText:
+                                                                  'Password',
+                                                              isPassword: true,
+                                                              onSaved: (String
+                                                                  value) {
+                                                                if (validator
+                                                                    .isLength(
+                                                                        value,
+                                                                        8)) {
+                                                                  //saves password to initialDetails object
+                                                                  initialDetails
+                                                                          .password =
+                                                                      value;
+                                                                }
+                                                                return null;
+                                                              },
+                                                              validator: (String
+                                                                  value) {
+                                                                //makes sure that the password is at least 8 characters long
+                                                                if (!validator
+                                                                    .isLength(
+                                                                        value,
+                                                                        8)) {
+                                                                  return 'Password must be at least 8 characters';
+                                                                }
+                                                                return null;
+                                                              },
+                                                            ),
+                                                          ],
+                                                        )))))))),
```

Comment:
Make sure to use trailing commas to prevent the formatter from de-tree-ifying the widget hierarchy.
```suggestion
                                                        ),
                                                      ),
                                                    ),
                                                  ),
                                                ),
                                              ),
                                            ),
                                          ),
```